### ISSUE TITLE
Require ToggleSwitch component to have a label

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#2036]
Currently the toggle switch does not have a label. Have it require an `aria-label` or `aria-labelledby` attribute.
